clippy_utils/src/attrs.rs

```diff
@@ -28,6 +28,7 @@ pub const BUILTIN_ATTRIBUTES: &[(&str, DeprecationStatus)] = &[
     ("dump",                  DeprecationStatus::None),
     ("msrv",                  DeprecationStatus::None),
     ("has_significant_drop",  DeprecationStatus::None),
```

Comment:
This and the new attribute are different from the others above: they are meant for crate authors, instead of for clippy users.

Can you add a comment about that, before this line, please? Also to separate them in our code.

Reply:
thx @flip1995, added docs for it too

Reply:
Can you add a comment in the code here please, saying the the following macros are for crate authors?